### ISSUE TITLE
Test calling internal functions inside Governance

### ISSUE THREAD

**Issue by ferittuncer** [kleros/kleros#100]
It should be possible to call the using a proposal execution.

**Comment by ferittuncer** [comment]
Try making internal function public and adding an onlyItself modifier. `executeProposal` is using `.call()` to execute proposals and it's an external call, which will fail on internal functions.

**Comment by epiqueras** [comment]
Make a test contract that inherits this one and exposes the functions.

**Comment by ferittuncer** [comment]
@epiqueras Problem is not about testing internal functions. Problem is it should be possible to create and execute proposals which call internal functions. `executeProposal()` is implemented using `.call()` so even Governance contract itself is the caller, can't call internal functions.

I verified that this is the case by testing it. Internal doesn't work but instead using `msg.sender == address(this)` works.

You previously told me that I can use `myContract.myMethod.getData()` but I couldn't figure it out how. Instead, I used `encodeFunctionCall()` with Truffle 5 in `demo/governance` branch, which demonstrates calling `setVotingTime()` function using a proposal execution. We can't merge it yet, but soon Truffle 5 will be stable and we will upgrade, so I think it's OK.
